cli/cmd/main.go

```diff
@@ -185,6 +207,92 @@ func main() {
 					},
 				},
 			},
+			// TODO: Add help stuff.
+			{
+				Name:  "data",
+				Usage: "download synced data",
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:     dataFlagDestination,
+						Required: true,
+						Usage:    "output directory for downloaded data",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagType,
+						Required: true,
+						Usage:    "data type to be downloaded: either binary or tabular",
+					},
+					&cli.StringSliceFlag{
+						Name:     dataFlagOrgs,
+						Required: false,
+						Usage:    "org filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagLocation,
+						Required: false,
+						Usage:    "location filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagRobotID,
+						Required: false,
+						Usage:    "robot_id filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagPartID,
+						Required: false,
+						Usage:    "part_id filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagRobotName,
+						Required: false,
+						Usage:    "robot_name filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagPartName,
+						Required: false,
+						Usage:    "part_name filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagComponentType,
+						Required: false,
+						Usage:    "component_type filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagComponentModel,
+						Required: false,
+						Usage:    "component model filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagComponentName,
+						Required: false,
+						Usage:    "component name filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagMethod,
+						Required: false,
+						Usage:    "method filter",
+					},
+					&cli.StringSliceFlag{
```

Comment:
Question: how are StringSlices parsed? just wondering how i should format values that are intended to be arrays

Reply:
You pass them as comma separated values
e.g.
`viam data --orgs 1c614556-2ff9-4234-9a94-d59b0a6d3378 --destination /tmp/cli_debug_test/2 --mime_types image/jpeg,image/png --type binary`